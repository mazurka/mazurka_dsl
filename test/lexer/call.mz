call()

other_module:func(Var, Var2)

whitespace(Var Var2 Var3)

@resource:call(Other)

@local_call(Test)