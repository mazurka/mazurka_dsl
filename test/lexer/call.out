{call_begin,1,call}.
{call_or_affordance_end,1}.
{call_begin,3,{other_module,func}}.
{variable,3,'Var'}.
{variable,3,'Var2'}.
{call_or_affordance_end,3}.
{call_begin,5,whitespace}.
{variable,5,'Var'}.
{variable,5,'Var2'}.
{variable,5,'Var3'}.
{call_or_affordance_end,5}.
{affordance_begin,7,{resource,call}}.
{variable,7,'Other'}.
{call_or_affordance_end,7}.
{affordance_begin,9,local_call}.
{variable,9,'Test'}.
{call_or_affordance_end,9}.